packages/fast-layouts-react/src/pane/pane.props.ts

```diff
@@ -24,7 +29,7 @@ export interface PaneHandledProps extends PaneManagedClasses {
     maxWidth?: number;
 
     /**
-     * The initial width of the pane
+     * The width of the pane (fixed)
```

Comment:
sorry - not quite related, but would you mind adding a note here that if this is less than `minWidth` or greater than `maxWidth` then the component will sizeto the min or max respectively?

Reply:
sure

Reply:
adding here and to initialWidth